bam.iobio/package.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "@overture-stack/bam-visualizations",
+	"version": "0.1.0",
+	"private": true,
+	"type": "module",
+	"dependencies": {
+		"@emotion/react": "^11.11.4",
```

Comment:
Emotion is source of bug w/ local Stage integration -- adding as dependency resolves the issue